examples/sensor-websocket/src/components/SensorData.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Alert, Col, Container, ListGroup, Row } from "react-bootstrap";
+
+function ListItem(props) {
+  return (
+    <ListGroup.Item
+      key={props.index}
+      className="temperature-list px-3 border-0 pb-0 py-2 font-weight-normal bg-transparent"
```

Comment:
forcing a font weight here is not a good idea. It makes ALL children components inherit this style.
Use it only in the components that require it